apps/editor.planx.uk/src/@planx/components/shared/RuleBuilder/utils.ts

```diff
@@ -1,11 +1,13 @@
 import { Store } from "pages/FlowEditor/lib/store";
 
-import { Condition, ConditionalRule, Operator, Rule } from "./types";
+import { Condition, Rule } from "./types";
+
+export const isRuleMet = (passport: Store.Passport, rule: Rule): boolean => {
+  // Simple (non-conditional) rules always evaluate to true
+  if (rule.condition === Condition.AlwaysRequired) return true;
+  if (rule.condition === Condition.AlwaysRecommended) return true;
+  if (rule.condition === Condition.NotRequired) return true;
```

Comment:
Slightly verbose, but checking each item allows TS to correctly type-narrow. Using something like `simpleConditions.includes(rule.condition)` won't work here to type-narrow.